source/fleetprovisioning/FleetProvisioning.h

```diff
@@ -124,6 +124,11 @@ namespace Aws
                      */
                     Aws::Crt::String templateName;
 
+                   /**
+                    * \brief stores Fleet Provisioning template parameters
```

Comment:
Can you expand on this description to provide an example of what this might be used for? As is the comment doesn't really tell more of a story than the class and variable names

Reply:
+1 
Please add more details over here and in the Readme file for users.